iina/Base.lproj/Localizable.strings

```diff
@@ -167,6 +167,12 @@
 "preference.sub_override_level.strip" = "strip";
 "preference.sub_override_level.yes" = "yes";
 
+"preference.sub_override_level.descriptive_text.no" = "No override: render subtitles as specified by the subtitle scripts.";
+"preference.sub_override_level.descriptive_text.force" = "Only allow override using the --sub-ass-* mpv options.";
+"preference.sub_override_level.descriptive_text.scale" = "Allow adjusting scale of ASS subtitles and override using the --sub-ass-* mpv options.";
+"preference.sub_override_level.descriptive_text.strip" = "Allow adjusting scale and style of ASS subtitles.";
+"preference.sub_override_level.descriptive_text.yes" = "Ignore all style styles from ASS subtitles and override them by IINA settings.";
```

Comment:
This is not what `yes` does. For `yes` to make sense we need to implement support for a couple of the `--sub-ass-*` styles, putting those controls in the `ASS Subtitles` section. Then we can say something like:
`Apply all the style settings in this section.`